examples/src/utils/InspectNodeUi.ts

```diff
@@ -55,11 +59,41 @@ export class InspectNodeUI {
       hide: () => {
         const nodeCollection = new TreeIndexNodeCollection(new NumericRange(node.treeIndex, node.subtreeSize));
         model.assignStyledNodeCollection(nodeCollection, DefaultNodeAppearance.Hidden);
+      },
+      addTranslationGizmo: async () => {
+        // Create dummy object that can be dragged around
+        const bbox = await model.getBoundingBoxByTreeIndex(node.treeIndex);
+        const startPosition = bbox.getCenter(new THREE.Vector3());
+        const draggable = new THREE.Object3D();
+        draggable.position.copy(startPosition);
+        
+        const gizmo = new TransformControls(this._viewer.getCamera(), this._viewer.renderer.domElement);
+        const offset = new THREE.Vector3();
+        const transform = new THREE.Matrix4();
+        gizmo.addEventListener('change', () => {
+          offset.copy(draggable.position);
+          offset.addScaledVector(startPosition, -1.0);
+          transform.makeTranslation(offset.x, offset.y, offset.z);
+          model.setNodeTransformByTreeIndex(node.treeIndex, transform);
+          console.log(node.treeIndex,  offset, transform.clone());
```

Comment:
Remove this, creates insane amount of spam in the console.

Reply:
oops